sqlalchemy_redshift/commands.py

```diff
@@ -158,7 +163,7 @@ class UnloadFromSelect(_ExecutableClause):
     def __init__(self, select, unload_location, access_key_id=None,
                  secret_access_key=None, session_token=None,
                  aws_account_id=None, iam_role_name=None,
-                 manifest=False, delimiter=None, fixed_width=None,
+                 manifest=False, header=None, delimiter=None, fixed_width=None,
```

Comment:
It would be safer to add this new param to the end of the list in case anyone is relying on calling this with only positional arguments.

Reply:
Also, would be good to have this default to `False` like `manifest` so we have a pure boolean rather than having to think about the `None` case.